app/src/components/DataPanel.vue

```diff
@@ -645,6 +671,11 @@ export default {
       this.$store.commit('indicators/CUSTOM_AREA_INDICATOR_LOAD_FINISHED', null);
       refMap.onResize();
     },
+    generateChart() {
+      this.previousArea = { ...this.selectedArea };
```

Comment:
I think this is not needed anymore

Reply:
Please elaborate - this function is important, chart (re)generation does not work without it.